### DISCUSSION TITLE
Its possible to use to_tsquery on fulltext

### DISCUSSION THREAD

**Discussion by renatogabrielbr** [mikro-orm/mikro-orm#4467] [General]
Hello, I want to know if its possible some how, without use raw sql, to use to_tsquery instead of plainto_tsquery ?
I think plainto_tsquery doesnt support wildcard. ( postgresql)

`field : {$fulltext : 'ab:*}`

Thanks

**Comment by B4nan** [comment]
cc @jsprw, not sure if this is covered with your changes in v6 too?

**Reply by jsprw** [reply]
It's possible by what's in the v6 branch now. However, I've started working on what I described [here](https://github.com/mikro-orm/mikro-orm/pull/3805#issuecomment-1408461872) and that would include support for this use-case. I hope to finish it for all platforms mid-july.

**Reply by B4nan** [reply]
Great to hear that! I hope we could have the v6 branch merged to master by that time, let's see, I still have some refactorings I want to undergo first, and I haven't touched the docs much either.

**Comment by renatogabrielbr** [comment]
Hello thanks for the answer both, I`m using v6.0.0-dev.87, so i can use it already?

**Comment by ognjenkljajic-huspy** [comment]
@B4nan any news on this?
